<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnHtml">
       <property name="text">
        <string>html</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImage">
       <property name="text">
        <string>image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnText">
       <property name="text">
        <string>text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUrl">
       <property name="text">
        <string>url</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnColor">
       <property name="text">
        <string>color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
